src/components/custom/CustomValueContainer.js

```diff
@@ -11,9 +11,9 @@ const CustomValueContainer = ({ children, ...props }) => {
 
   // values is an array of objects corresponding to the currently made selection
   let [values] = children;
-  const { length } = values;
 
-  if (length) {
+  if (Array.isArray(values)) {
```

Comment:
Won't values always be an array given the deconstruction in line 13?

Reply:
So this is the issue... when you *select* name(s) from the dropdown, `children = [//array of objects corresponding to selections made, some react-select config object]`, and so `[values] = children` means `values = [//array of objects corresponding to selections made]`. *however*, when you try *typing* into the dropdown, `children = [null, react-select config object]`, so `values = null`, and thus `{ length } = values` crashes the app... (you can try this on the live version 😐).

Reply:
OK. Got it. Maybe add the comment to explain this nuance.